qutebrowser/components/misccommands.py

```diff
@@ -339,16 +339,23 @@ def debug_webaction(tab: apitypes.Tab, action: str, count: int = 1) -> None:
 
 @cmdutils.register()
 @cmdutils.argument('tab', value=cmdutils.Value.count_tab)
-def tab_mute(tab: Optional[apitypes.Tab]) -> None:
+@cmdutils.argument('to_set', choices=['true', 'false'])
```

Comment:
`to-set` is a bit of a strange name from an users perspective. Perhaps simply `mute`, or `muted`, or `state`?

Reply:
Makes sense! I'll go with ``mute``.